### PR TITLE
test getFramebufferAttachmentParameter with stencil option enabled/di…

### PR DESCRIPTION
**Pull request by howareyou322**

### CODE REVIEW

**Comment by howareyou322** [comment]
This patch is trying to fix issue #1426

File: sdk/tests/js/tests/gl-object-get-calls.js [hunk 1]
```diff
@@ -107,186 +91,221 @@ testInvalidArgument(
...
+];
+
+for (var run = 0; run < testCases.length; ++run) {
+  debug("");
+  debug("Test getFramebufferAttachmentParameter with stencil " + testCases[run].contextStencil);
+
+  if (testCases[run].contextStencil) {
+    gl = wtu.create3DContext(null, {stencil: true});
+  } else {
+      gl = wtu.create3DContext(null, {stencil: false});
```
Comment (zhenyao) [inline sdk/tests/js/tests/gl-object-get-calls.js]: wrong indent

**Comment by zhenyao** [comment]
Mostly looks good.  In theory you only need to test the default fbo with stencil on/off.  The client fbo isn't affected, but that's not a big deal. Merging now.  Please submit another CL to fix the minor indent issue.

File: sdk/tests/js/tests/gl-object-get-calls.js (hunk 1 above)
Comment (howareyou322) [inline sdk/tests/js/tests/gl-object-get-calls.js]: From line 94, it is the beginning of getFramebufferAttachmentParameter test case. Therefore, there are no indent in line 94, just like other test cases.

Comment (zhenyao) [inline sdk/tests/js/tests/gl-object-get-calls.js]: I am talking about line 106

**Pull request merged**